go/billing/tests/test_api.py

```diff
@@ -259,6 +259,82 @@ def test_low_credit_notification_zero_last_topup_value(self):
         self.assertFalse(mock_task_delay.called)
 
     @inlineCallbacks
+    def test_credit_cutoff_inbound(self):
+        self.patch(app_settings, 'ENABLE_LOW_CREDIT_CUTOFF', True)
+
+        mk_message_cost(
+            tag_pool=self.pool1,
+            message_direction=MessageCost.DIRECTION_INBOUND,
+            message_cost=0.2,
+            storage_cost=0.0,
+            session_cost=0.0,
+            markup_percent=0.0)
+
+        load_account_credits(self.account, 10)
+
+        transaction1 = yield self.create_api_transaction(
+            account_number=self.account.account_number,
+            message_id='msg-id-1',
+            tag_pool_name='pool1',
+            tag_name='tag1',
+            message_direction=MessageCost.DIRECTION_INBOUND,
+            session_created=False,
+            transaction_type=Transaction.TRANSACTION_TYPE_MESSAGE)
+
+        self.assertFalse(transaction1.get('credit_cutoff_reached', False))
+        self.assertEqual(Transaction.objects.count(), 2)
+
+        transaction2 = yield self.create_api_transaction(
+            account_number=self.account.account_number,
+            message_id='msg-id-1',
+            tag_pool_name='pool1',
+            tag_name='tag1',
+            message_direction=MessageCost.DIRECTION_INBOUND,
+            session_created=False,
+            transaction_type=Transaction.TRANSACTION_TYPE_MESSAGE)
+
+        self.assertTrue(transaction2.get('credit_cutoff_reached', False))
```

Comment:
Ah! I see.

I think two tests would be better, but I misread the `.assertTrue` as an
`.assertEqual` because of the `, False` at the end (but I see now it's part
of the `.get()`) call.